src/tag.rs

```diff
@@ -1358,6 +1360,27 @@ impl<'a> Tag {
                 }
             })
     }
+
+    /// Reads AIFF file and returns ID3 Tag from it
+    pub fn read_from_aiff(path: impl AsRef<Path>) -> crate::Result<Tag> {
```

Comment:
I am not sure how I feel about adding format specific reading and writing functions to the public API. Preferable, I would rather have a unified set of functions if feasible. I'll give this some thought and get back to you

Reply:
Up to you, I think it should be separate, due to how different it is.